PKHeX/Saves/SAV1.cs

```diff
@@ -328,7 +328,7 @@ public uint Coin
         {
             get
             {
-                return uint.Parse(BigEndian.ToUInt16(Data, Japanese ? 0x2846 : 0x2850).ToString("X4"));
+                return Convert.ToUInt16(BigEndian.ToUInt16(Data, Japanese ? 0x2846 : 0x2850).ToString("X4"), 16);
```

Comment:
I'm probably missing something, but why not just return `BigEndian.ToUInt16`?, instead of turning it to a string then converting it back?